jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/resources/i18n/Constants.java

```diff
@@ -98,4 +98,14 @@ String DataSetError(String dataSet,
     String Custom();
 
     String Home();
+
+    String Filters();
+
+    String Select();
+
+    String Refine();
```

Comment:
Is this the word UX team went for in the end? Doesn't sounds too descriptive, doesn't it? :)
Even _Filter by_ looks clearer to me.

Reply:
+1, we can discuss that with Liz, it is in the design and prototype that I'm using as the reference.